examples/chart/teleport/templates/config.yaml

```diff
@@ -66,26 +66,34 @@ data:
 {{ toYaml .Values.config.teleport.ssh_service.enhanced_recording | indent 8 }}
       pam:
 {{ toYaml .Values.config.teleport.ssh_service.pam | indent 8 }}
-      
+
     proxy_service:
       enabled: {{ .Values.config.teleport.proxy_service.enabled }}
-      public_addr: {{ .Values.config.public_address }}:{{ .Values.ports.proxyweb.containerPort }}
+{{- if .Values.ingress.enabled }}
+      {{- if .Values.ingress.tls }}
+      public_addr: {{ .Values.config.public_address }}:443
```

Comment:
Assumptions:
- when TLS is configured on the `Ingress` it'll be using port 443
- when TLS is not configured on the `Ingress` it'll be using port 80

When not using an `Ingress`, we change to using the host-side `proxyweb` port under the service definition rather than the container-side port.